src/common/transformations/include/transformations/utils/utils.hpp

```diff
@@ -193,7 +193,9 @@ TRANSFORMATIONS_API bool constantIsEqualTo(const std::shared_ptr<ov::op::v0::Con
 
 TRANSFORMATIONS_API bool has_f16_constants(const std::shared_ptr<const ov::Model>& function);
 
-TRANSFORMATIONS_API bool is_large_language_model(const ov::Model& model);
+TRANSFORMATIONS_API bool is_large_language_model(
+    const ov::Model& model,
+    std::function<bool(std::shared_ptr<ov::Node>)> custom_condition = nullptr);
```

Comment:
it's better to have a default binding which always returns false instead of nullptr

Reply:
Updated with return false by default